configure.py

```diff
@@ -323,6 +323,8 @@ def binary(name):
               # Disable size_t -> int truncation warning.
               # We never have strings or arrays larger than 2**31.
               '/wd4267',
+              # Disable warnings about narrowing conversions
+              '/wd4838', '/wd4309',
```

Comment:
I discourage this change.

Reply:
Removed and cleaned up corresponding code.